ppl/detail/util/formatDur.ts

```diff
@@ -0,0 +1,38 @@
+import {intervalToDuration} from "date-fns"
+
+const units = ["years", "months", "days", "hours", "minutes", "seconds"]
+
+// Takes two dates and returns the two most significant parts of their duration
+export default function formatDir(start, end) {
+  if (!start || !end) return "Not available"
+  const obj = intervalToDuration({start, end})
+  let allZero = true
+  let parts = []
+  for (const unit of units) {
+    if (allZero && obj[unit] === 0) continue
+    allZero = false
+    parts.push(formatPart(obj[unit], unit))
+  }
+  if (allZero) {
+    parts.push(formatPart((end.getTime() - start.getTime()) / 1000, "seconds"))
+  }
+  return firstTwo(parts).join(" ")
+}
+
+function formatPart(amount, unit) {
+  if (amount === 1) return [amount, singular(unit)].join(" ")
+  else return [amount, unit].join(" ")
+}
+
+function singular(unit) {
+  return unit.replace(/s$/, "")
+}
+
+function firstTwo(array) {
+  const first = []
+  for (let i = 0; i < array.length; ++i) {
+    if (i === 2) break
+    first.push(array[i])
+  }
+  return first
+}
```

Comment:
This function allows for nice, human readable durations. All the other date formatting functions in date-fns and momentjs were not human friendly enough for our case. 

Instead of choosing one unit, this function takes the two most significant units and displays them. For example:

```
2 years, 3 months
2 months, 5 days
1 day, 3 hours
2 hours, 59 minutes
1 minute, 3 seconds
and finally
54.2 seconds
0.34 seconds
```